packages/coding-standards/tests/Unit/CsFixer/Phpdoc/NoUselessAccessFixer/wrong/Simple.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\CodingStandards\Unit\CsFixer\Phpdoc\NoUselessAccessFixer;
+
+class Simple
+{
+    /**
+     * @access private
+     */
+    public const A = 'value';
```

Comment:
That's right and it's intentional (that's why the test is like this).
In my opinion, visibility should always have a higher priority because a code directly depends on it (bad visibility break application, bad annotation don't do anything).

Access annotation is merely a hack, to achieve backward compatibility and it's said in PR description:

> Every constant have to have visibility defined with the access modifier or with @access annotation.
The first is recommended and in the future, the access annotation will not be present in the code at all.

Reply:
@access is a user, check that link :-D

Reply:
If it's intentional, I would :heart: to have it expresses somewhere in the code. Maybe in the test scenario name? Maybe in commit message?

Reply:
I created another test file, so it's covered a simple use-case and mixed access and visibility.
I add the note to commit message on the rebase